.github/workflows/lint_and_test.yml

```
@@ -0,0 +1,23 @@
+name: lint_and_test
+on: push
+
+jobs:
+  lint:
+    name: Run linter
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+      - run: bundle install --jobs 4 --retry 3
+      - run: bin/lint
+
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+      - run: bundle install --jobs 4 --retry 3
+      - run: bin/test
```

In the old CircleCI config, we were uploading the individual test results as CI artifacts:
https://github.com/mycase/boatload/blob/425534219a7eb725c977cced12f118dbf882ff9e/.circleci/config.yml#L32-L33

Do we want to do that here, too?

Or, do we want to remove `minitest-ci` and run minitest verbosely to include test timings in the CI log?
```suggestion
      - run: bin/test
        env:
          TESTOPTS: --verbose
```
 (This would give a bit less granular data per test than the XML test reports, but might be enough for our liking, while avoiding saving extra artifacts.)